### PR TITLE
add low-power events experimental extension spec

### PR DESCRIPTION
**Pull request by pbalcer**

### CODE REVIEW

**Review by kbenzie — COMMENTED** [review]
The spec file seems to be missing.

**Comment by pbalcer** [comment]
> The spec file seems to be missing.

:| done.

**Review by kbenzie — APPROVED** [review]

File: include/ur_api.h [hunk 1]
```diff
@@ -1697,6 +1699,7 @@ typedef enum ur_device_info_t {
                                                                      ///< backed 2D sampled image data.
     UR_DEVICE_INFO_ENQUEUE_NATIVE_COMMAND_SUPPORT_EXP = 0x2020,      ///< [::ur_bool_t] returns true if the device supports enqueueing of native
                                                                      ///< work
+    UR_DEVICE_INFO_LOW_POWER_EVENTS_EXP = 0x2021,                    ///< [::ur_bool_t] returns true if the device supports low-power events.
```
Comment (steffenlarsen) [inline include/ur_api.h:1702]: I am not against having this query, but it is not strictly necessary, given the property is a hint, so backends can just ignore it. No harm in it though!

Comment (pbalcer) [reply include/ur_api.h:1702]: I added it mostly for consistency sake. All the other experimental extensions add a query like that so 🤷

**Review by nrspruit — APPROVED** [review]
LGTM for L0

**Comment by pbalcer** [comment]
@oneapi-src/unified-runtime-native-cpu-write please review

**Comment by pbalcer** [comment]
@oneapi-src/unified-runtime-native-cpu-write ping

**Comment by pbalcer** [comment]
@oneapi-src/unified-runtime-native-cpu-write ping

**Comment by pbalcer** [comment]
@oneapi-src/unified-runtime-native-cpu-write ping

**Review by PietroGhg — APPROVED** [review]

**Pull request merged**